content/docs/enterprise/external-data/geoip.mdx

```diff
@@ -1,47 +1,100 @@
 ---
-title: GeoIP Ranges
-draft: true
-description: Configure the $EXT_DATA_PROVIDER external data provider to extend your access policies.
+title: GeoIP
+description: Configure the GeoIP external data provider to extend your access policies.
 ---
 
-import Tabs from '@theme/Tabs';
-import TabItem from '@theme/TabItem';
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
 
-The $EXT_DATA_PROVIDER integration provides {{Fill me in with content specific to this data provider}}.
-
-This dataset provides a list of IP address ranges and their country of origin, according to ARIN:
+The GeoIP integration provides provides a list of IP address ranges and their country of origin:
 
 ```json tittle="Example"
-{"$index":{"cidr":"1.0.0.0/24"},"id":"1.0.0.0/24","country":"US","state":"","city":"","zip":"","timezone":""},
-{"$index":{"cidr":"1.0.1.0/24"},"id":"1.0.1.0/24","country":"CN","state":"","city":"","zip":"","timezone":""},
+[
+  {
+    "$index": { "cidr": "1.0.0.0/24" },
+    "id": "1.0.0.0/24",
+    "country": "US",
+    "state": "",
+    "city": "",
+    "zip": "",
+    "timezone": ""
+  },
+  {
+    "$index": { "cidr": "1.0.1.0/24" },
+    "id": "1.0.1.0/24",
+    "country": "CN",
+    "state": "",
+    "city": "",
+    "zip": "",
+    "timezone": ""
+  }
+]
 ```
 
-
 ## Install
 
+<Tabs>
+<TabItem value="compose" label="Docker Compose">
+
 These instructions assume a local testing environment using [Docker Compose]. Adjust as needed for your deployment environment.
 
+1. Download the IP2Location CSV GeoIP database from [ip2location.com].
+
+1. Add the datasource docker image to Docker Compose:
+
+```yaml title="docker-compose.yaml"
+services:
+  ip2location:
+    image: docker.cloudsmith.io/pomerium/datasource/datasource:main
+    container_name: ip2location
+    command:
+      - ip2location
+      - /var/ip2location.csv.zip
+    ports:
+      - 8080:8080
+    volumes:
+      - ${PWD}/IP2LOCATION-LITE-DB1.CSV.ZIP:/var/ip2location.csv.zip
+```
 
+1. Bring up the new container.
 
-<Tabs>
-<TabItem value="cloudsmith" label="Cloudsmith">
+</TabItem>
+<TabItem value="binary" label="Local Binary">
 
+1. Download the latest [release] of the [Pomerium datasource] project and extract it.
 
+1. Change directory and run the binary:
 
-</TabItem>
-<TabItem value="dockerhub" label="DockerHub">
+  ```sh
+  cd datasource-v*
+  ./datasource ip2location ../IP2LOCATION-LITE-DB1.CSV.ZIP
+  ```
 
+  The output should resemble:
 
+  ```sh
+  <nil> INF starting ip2location http server address=:8080 file=../IP2LOCATION-LITE-DB1.CSV.ZIP
```

Comment:
@calebdoxsey Should users expect this output to update when console pulls new data?

Reply:
Answer: likely not.